### PR TITLE
edited the Wizard 

### PR DESCRIPTION
**Pull request by sonejah21**
removes disabled buttons in footer for cleaner intro and finish using "ng-switch-when"; added intro message and congratulations message

### CODE REVIEW

File: src/client/app/components/wizard/wizard-modal.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
 </div>
 <div class="wizard-modal__body" ng-switch="vm.state">
   <div ng-switch-when="intro">
-    <button type="button" class="btn-rounded wizard-modal__button" ng-click="vm.startWizard()">Start Wizard</button>
+    <div class="wizard-modal__question">Welcome to the Wizard!</div>
+    <div class="wizard-modal__subtitle">Determine the best-suited products for your project<br>by answering six simple questions.</div>
```
Comment (stackus) [inline src/client/app/components/wizard/wizard-modal.html]: The number of questions is variable. We cannot display a count here in the text.

Comment (sonejah21) [inline src/client/app/components/wizard/wizard-modal.html]: Yeah, I used the styling from the "wizard-modal__question" - didn't want it to necessarily act like a question, just match the same font size/style as the questions. I will duplicate the styling and give it a different name.

Comment (stackus) [inline src/client/app/components/wizard/wizard-modal.html]: My comment was about this `by answering six simple questions`. Your reply about styling confuses me.

Comment (sonejah21) [inline src/client/app/components/wizard/wizard-modal.html]: Oh that! Got it. I totally misunderstood you. Changing the correct part.

**Pull request merged**